tests/apps/query_performing_app/apps.py

```diff
@@ -0,0 +1,56 @@
+from django.apps import AppConfig
+from django.db import connections
+
+
+class BaseAppConfig(AppConfig):
+    name = "apps.query_performing_app"
+    database = "default"
+    use_model = True
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.query_results = []
+
+    def ready(self):
+        self.query_results = []
+
+        if self.use_model:
+            self._perform_query_using_model()
+        else:
+            self._perform_query_using_cursor()
+
+    def _perform_query_using_model(self):
+        from ..models import TotallyNormal
+
+        queryset = TotallyNormal.objects.using(self.database)
+        queryset.create(name=self.label)
+        self.query_results = list(queryset.filter(name=self.label))
+
+    def _perform_query_using_cursor(self):
+        from ..models import TotallyNormal
+
+        with connections[self.database].cursor() as cursor:
+            db_table = TotallyNormal._meta.db_table
+            cursor.execute(f"INSERT INTO {db_table}(name) VALUES (%s)", [self.label])
+            cursor.execute(f"SELECT name FROM {db_table} WHERE name = %s", [self.label])
+            self.query_results = cursor.fetchall()
```

Comment:
It's not necessary to fetch data from the existing table, so I'd simplify this with bare select
```suggestion
        connection = connections[self.database]
        with connection.cursor() as cursor:
            cursor.execute("SELECT 2" + connection.features.bare_select_suffix)
            self.query_results = cursor.fetchone()
```